scss/partials/homeKoya.scss

```diff
@@ -0,0 +1,381 @@
+                color: variables.$testimonial-bk-color;
+                border: none;
+            }
+        }
+    }
+}
+
+@media (max-width: 768px) {
+    .home_makeup {
+        margin: 0;
+        padding: 0;
+
+        .home_makeup_title {
+            font-size: 2.5rem;
+            margin: 0;
+            padding: 2rem 0;
+        }
+
+        .home_makeup_content {
+            margin: 0;
+            display: flex;
+            flex-direction: column;
+
+            .cards {
+                .card {
+                    flex-direction: column;
+                    margin: 0;
+
+                    .card_left {
+                        .card_title {
+                            flex-direction: column;
+                            text-align: center;
+                        }
+
+                        .card_description {
+                            text-align: center;
+                        }
+                    }
+
+                    .view_class_button {
+                        display: flex;
+                        justify-content: center;
+                        margin: 1rem 0;
+
+                        .view_class {
+                            display: block;
+                            margin-right: 1rem;
+                            font-size: 1.25rem;
+                        }
+
+                        .vector {
+                            margin: 0;
+                        }
+                    }
+                }
+            }
+
+            .home_makeup_button {
+                margin: .75rem 30%;
+            }
+
+            .makeup_image {
+                max-width: none;
+                width: 100%;
+            }
+        }
+    }
+}
+
+.home_diy_beauty {
+    margin-top: 4rem;
+
+    .home_diy_beauty_separator {
+        display: none;
+    }
+
+    .home_diy_title {
+        font-size: 3rem;
+        text-align: center;
+        color: variables.$text-darkbrown;
+    }
+
+    .diy_beauty_corner_images {
+        display: flex;
+        justify-content: center;
+        margin: 3rem 8rem;
+        width: auto;
+        text-align: center;
+
+        img {
+            max-width: 18.625rem;
+            margin: 1rem 2rem;
+            transition: filter 0.3s ease-in-out;
+
+            &:hover {
+                filter: brightness(0.5);
+            }
+        }
+    }
+}
+
+.diy_beauty_button {
+    display: inline-block;
+    border: 1px solid variables.$text-darkbrown;
+    padding: 1rem 1.5rem;
+    margin: .75rem 45%;
+    font-size: 1.25rem;
+    background-color: variables.$primary-color;
+    text-align: center;
+
+    &:hover {
+        background-color: variables.$dark-pink;
+        color: variables.$testimonial-bk-color;
+        border: none;
+    }
+}
+
+@media (max-width: 768px) {
+    .home_diy_beauty {
+        .diy_beauty_corner_images {
+            display: none;
+        }
+    }
+}
+
+.slide-wrap {
+    display: flex;
+    margin: 3rem 29%;
```

Comment:
I'm not sure if it's here or another line, but when the screen is large, these images are aligned to the right, not the center. Could you check it?"